src/instructlab/utils.py

```diff
@@ -833,3 +833,50 @@ def clear_directory(path: pathlib.Path) -> None:
     if path.exists():
         shutil.rmtree(path)
     os.makedirs(path)
+
+
+def wrap_text_to_max_length(text: str, max_line_length: int = 80) -> str:
```

Comment:
This function is pretty cool but i'm wondering if we can use something like the `textwrap` module do handle some of this instead of rewriting it? not 100% sure but i think it should be native to python

Reply:
Done thanks!